falcon/response.py

```diff
@@ -123,6 +133,103 @@ def set_stream(self, stream, stream_len):
         self.stream = stream
         self.stream_len = stream_len
 
+    def set_cookie(self, key, value, **cookie_params):
+        """Set a response cookie.
+
+        Args:
+            key (str): Cookie key name
+            value (str): Cookie value
+
+        Kwargs:
+            path (str): The Path attribute specifies the subset of URLs to
+                which this cookie applies.
+            comment (str): Because cookies can contain private information
+                about a user, the Cookie attribute allows an origin server to
+                document its intended use of a cookie. The user can inspect
+                the information to decide whether to initiate or continue a
+                session with this cookie.
+            domain (str): The Domain attribute specifies the domain for
+                which the cookie is valid.
+                An explicitly specified domain must always start with a dot.
+            max_age (int): The Max-Age attribute defines the lifetime of the
+                cookie, in seconds.  The delta-seconds value is a decimal non-
+                negative integer.  After delta-seconds seconds elapse,
+                the client should discard the cookie.
+                A value of 0 means the cookie should be discarded immediately.
+            expires (str): Defines when the cookie should expire.
+                the default is to expire when the browser is closed.
+                The format given should be "Wdy, DD Mon YYYY HH:MM:SS GMT"
+            secure (bool): The Secure attribute directs the user
+                agent to use only secure means to contact the origin
+                server whenever it sends back this cookie.
+            version (str): The Version attribute, a decimal integer,
+                identifies to which version of the state management
+                specification the cookie conforms.
+            httponly (bool): The attribute httponly specifies that the cookie
```

Comment:
OK, maybe we can get some others from the community to chime in. It seems like the target audience for Falcon will usually want to enable `httponly` and `secure`, but that's just a guess.